src/platform/forms/tests/save-in-progress/SaveInProgressIntro.unit.spec.jsx

```diff
@@ -697,7 +697,7 @@ describe('Schemaform <SaveInProgressIntro>', () => {
         prefillsAvailable: [],
       },
       login: {
-        currentlyLoggedIn: true,
+        currentlyLoggedIn: false,
```

Comment:
It was accidentally left like that while I was testing... The test only passes because the `inProgress` message is set as an empty string, otherwise the alert is still shown. I wasn't really sure what this test was doing - testing the error message or that the alert should not be showing.